tests/testsuite/freshness.rs

```diff
@@ -2934,7 +2934,7 @@ fn use_mtime_cache_in_cargo_home() {
     p.cargo("check -v")
         .env("CARGO_HOME", &cargo_home)
         .with_status(101)
-        .with_stderr(
+        .with_stderr_contains(
```

Comment:
I know 79a7bf4 didn't fix these regressions the latest #14039 way. However, to avoid introducing more unrelated diff (redaction, larger snapshot, etc.) to this PR, I thought minimum fix might be more suitable.